src/components/WelcomeForm/WelcomeForm.jsx

```diff
@@ -0,0 +1,37 @@
+import styles from "./WelcomeForm.module.css";
+import Message from "./Message";
+import { useState } from "react";
+
+const WelcomeForm = () => {
+  const [user, setUser] = useState({ userName: "", age: "" });
+
+  const handleSubmit = (e) => {
+    const { userName, age } = e.target;
+    setUser({
+      userName: userName.value,
+      age: age.value,
```

Comment:
Then here you can use the parse method `parseInt(age.value)` to convert from string to number type